base/abstractarray.jl

```diff
@@ -2434,9 +2441,8 @@ function hash(A::AbstractArray, h::UInt)
     n = 0
     while true
         n += 1
-        # Hash the current key-index and its element
-        elt = A[keyidx]
```

Comment:
PkgEval notes that `elt` is used later in this function and is now causing UndefVarError in some packages last night. See for example:

https://s3.amazonaws.com/julialang-reports/nanosoldier/pkgeval/by_date/2021-03/17/DataStructures.1.7.0-DEV-d7da2a4a32.log

Reply:
oh. Good catch. Will make a new PR to fix this.